### ISSUE TITLE
Review LOI: High Reward

### ISSUE THREAD

**Issue by smiths** [smiths/TCTvMDE#8]
The first draft of the "high reward" page for the LOI is now complete (https://github.com/smiths/TCTvMDE/blob/master/LOI/HighReward.tex).  Please review.  Feel free to make changes directly in the tex source, or to suggest changes as comments in the discussion below.  I have embedded specific review requests by name in the document.  When you are done your review, please check your name off on the following list:

- [ ] @JacquesCarette 
- [ ] @richpaige 
- [x] @ZMotamed 

The instructions for this one page document are given below.

Describe the potential outcomes of the project. Provide a rationale for why they should be considered high reward.  "High reward" evaluation criteria:
- broad impact: having a high degree of impact and/or affecting a large community.
- reach: having a high degree of impact and/or affecting a large community.
- impact on research: significantly advancing current knowledge, methods and/or technologies
- transforming and/or disrupting conventional thinking;
- resolving a longstanding issue or debate;

**Comment by ZMotamed** [comment]
Done.

**Comment by smiths** [comment]
I'm reopening this issue because we aren't fully done the review.

**Comment by smiths** [comment]
Thank you for your changes @ZMotamed.  They look good.  Can you please e-mail me the page from your NSERC DG where you list the references?  I need to complete the references properly, and since I borrowed some text from your DG (about the impact of cardiovascular disease on the world and Canada).

**Comment by ZMotamed** [comment]
I just emailed the file to you.

**Comment by smiths** [comment]
Yes, @JacquesCarette there are more recent papers about the state of practice for SC software, like JohansonAndHasselbring2018 and Storer2017.  I cited the Kelly paper because she was the first, as far as I know, to highlight the gap between SE and SC.  (The original wording of the sentence mentioned the chasm between SE and SC, which is why Kelly was cited.  Your edits changed the meaning of the original sentence to the point where the citation really isn't accurate anymore.  Kelly doesn't even use the word documentation in her paper.  😄 )

Your point is well taken.  I will incorporate a newer reference.

**Comment by smiths** [comment]
@JacquesCarette, in response to your above comment:

> I am quite wary of mentioning " climate modelling, automotive safety" as future domains, as it seems to me that they fail the "well understood" test where we claim to be applicable. It may well be that very soon that understanding will be there, but I'm unsure if it's the case right now. [I won't change the text with that yet, but unless I'm convinced otherwise, will do so in another pass].

I think the divergence point for us is that I'm referring to the physics the models are based on, not the models themselves.  The cardiovascular system is also not well understood.  Nobody (that I am aware of) has a full 3D, time-dependent, model of the cardiovascular system.  However, the physics it is based on is well understood: conservation of mass, conservation of momentum, Newton's law of viscosity, etc.  The same kind of physics is applied for climate modelling and other domains.  How the models should be combined, what assumptions should be made, etc, are not well understood, but the the scientific underpinnings go back to Newton and his contemporaries.

If we take "well understood" too far, then we are left with software that solves projectile motion, and other simple examples.

If we don't select examples like climate modelling, what would be another tactic for highlighting high reward?
